settings/index.js

```diff
@@ -15,7 +15,7 @@ class MyProfile extends Component {
     this.pages = [
       {
         route: 'change-password',
-        label: this.props.stripes.intl.formatMessage({ id: 'ui-myprofile.settings.index.change.password' }),
+        label: this.props.stripes.intl.formatMessage({ id: 'ui-myprofile.settings.changePassword' }),
```

Comment:
Thanks for making these changes. The only one I am not sure about is losing the `index` part of the translation-key. Did you originally get that by copying some other bit of code? If so, then consistency may be important here, even though _to me_ it looks redundant.